Adds Vendata.org challenge (#34)
code-for-venezuela/2019-april-codeathon: challenges/VED-DEF/README.md

# Systematize Data about Defense Spending (Annual Report and Accounts)

## Who we are
Vendata is an organization with the mission of generating the largest collection of open data for information of public interest in Venezuela, through compilation, processing and transformation of data from closed to open formats.

During the first phase of the project we selected a sample documents and we defined the public interest topics that needed to be systematized in the database. Later, we established a basic structure and developed a web form of input. Each datum entered was checked and cleaned up.

> With the information available we conducted searches in the public records (Gaceta Oficial) regarding public enterprises, and annual reports as well as the investigations available in Vendata.org in partnership media sources and NGOs. The technology used for the creation of the web forms and the searched was PHP and SQL with complements from Java and Jquery.

Reviewer: ```suggestion
With the information available we conducted searches in the public records (Gaceta Oficial) regarding public enterprises, and annual reports as well as the investigations available in Vendata.org in partnership media sources and NGOs. The technology used for the creation of the web forms and the scraping was PHP and SQL with some smaller parts in Java and Jquery.
```

It has been more than two years since the government generates annual reports for the office of the public defender, comptroller and prosecutors). We are downloading and systematizing the annual report for the departments of environment, indigenous peoples, defense and mining.

We downloaded the tables that we found on the annual report of the Ministry of Defense for the years: 2011-2013-2014 and 2015. However we have not systematized and visualized this data.

## Data
The data that Vendata.org has collected is avaliable in this Google Drive folder: https://drive.google.com/drive/folders/1Xvcw4IxamCnWNAQutvowvymMX8v-3YLL?usp=sharing

## Challenge
 - Compile the data from different years in a single table so that we can analyze and visualize it
 - Determine which data can be visualized, some initial ideas are:
     - Distribution of military spending in the years covered, including investment in militias.
     - Completed projects: develop a search engine for all the projects of the Ministry of Defense during those years
     - Determine if there is additional and more current information on this topic available on the web.

## Skills needed

> Data Analysis, NLP, Data Visualization

Reviewer: I would remove NLP. Do we really need NLP here? (haven't looked that much at the data).

Author: Yes, I haven't either, let's remove it and add back if needed.
